week2/who_are_you/main.js

```diff
@@ -0,0 +1,99 @@
+import pic1 from "./assets/윈터.jpg";
+import pic2 from "./assets/조유리.jpg";
+import pic3 from "./assets/태연.webp";
+import pic4 from "./assets/장원영.jpg";
+import pic5 from "./assets/은하.jpg";
+
+const $ = (selector) => document.querySelector(selector);
+
+let currentStep;
+
+const quizList = [
+  {
+    src: pic1,
+    answer: "윈터",
+  },
+  {
+    src: pic2,
+    answer: "조유리",
+  },
+  {
+    src: pic3,
+    answer: "태연",
+  },
+  {
+    src: pic4,
+    answer: "장원영",
+  },
+  {
+    src: pic5,
+    answer: "은하",
+  },
+];
+
+function gameManager(gameInfo) {
+  initGame(gameInfo);
+  attachEvent(gameInfo);
+}
+
+window.onload = () => {
+  gameManager({
+    score: $('.scoreBoard__score'),
+    answer: $('ul.answer__list'),
+    image: $('.imageBoard > img'),
+  });
+}
```

Comment:
<다시하기 버튼 js로 만들기 튜토리얼1>
```suggestion
window.onload = () => {
  gameManager({
    score: $('.scoreBoard__score'),
    answer: $('ul.answer__list'),
    image: $('.imageBoard > img'),
    shuffleBtn: $('.buttonList__shuffle')
  });
}
```
gameManager에 버튼 정보도 함께 넘깁니다!